archive/generic-core-terms.md

```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Security is core to our values, and we value the input of hackers acting in good-faith to help us maintain a high standard for the security and privacy for our users. This includes encouraging responsible vulnerability research and disclosure. This policy sets out our definition of good-faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
```

Comment:
Remove the run on - '..., and...' after values.  Should be: Security is core to our values.  We value.....

Reply:
Not sure how to make more succinct.   "...This includes encouraging responsible vulnerability research and disclosure. This policy sets out our definition of good-faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return..."  Maybe rip it all out.  Then put in the item below: 

Something akin to the following - "....This policy sets out our definition of good-faith in the context of responsible vulnerability research and disclosure, as well as set expectations on what to expect in return....."

Reply:
@cloudriseup Curious re the second recommendation here: What's the rationale behind that?